src/library_wasm_worker.js

```diff
@@ -4,12 +4,15 @@
  * SPDX-License-Identifier: MIT
  */
 
+#if WASM_WORKERS == 2
+// Helpers for _wasmWorkerBlobUrl used in WASM_WORKERS == 2 mode
 {{{
   global.captureModuleArg = () => MODULARIZE ? '' : 'self.Module=d;';
   global.instantiateModule = () => MODULARIZE ? `${EXPORT_NAME}(d);` : '';
-  global.instantiateWasm = () => MINIMAL_RUNTIME ? '' : 'd[`instantiateWasm`]=(i,r)=>{var n=new WebAssembly.Instance(d[`wasm`],i);r(n,d[`wasm`]);return n.exports};';
+  global.instantiateWasm = () => MINIMAL_RUNTIME ? '' : 'd[`instantiateWasm`]=(i,r)=>{var n=new WebAssembly.Instance(d[`wasm`],i);return r(n,d[`wasm`]);};';
```

Comment:
Do we not need `.exports` here?

Reply:
The `r` argument to `instantiateWasm` which is the `recieveInstance` function takes the exports and can optionally wrap them, afterwhich it return the new exports object.

Without this change the workers we never seeing the wrapped exports.